packages/mobile/src/onboarding/registration/EnableBiometry.tsx

```diff
@@ -72,10 +77,13 @@ export default function EnableBiometry({ navigation, route }: Props) {
   }, [navigation, step, totalSteps])
 
   const onPressUseBiometry = async () => {
-    dispatch(setUseBiometry(true))
-    // do some stuff to use biometry
-
-    navigate(choseToRestoreAccount ? Screens.ImportWallet : Screens.VerificationEducationScreen)
+    try {
+      await setPincodeWithBiometrics()
+      dispatch(setPincodeSuccess(PincodeType.PhoneAuth))
+      navigate(choseToRestoreAccount ? Screens.ImportWallet : Screens.VerificationEducationScreen)
+    } catch (error) {
+      Logger.warn(TAG, 'Error enabling biometrics', error)
```

Comment:
I've chosen `warn` over `error` for most of the logging because the user action of cancelling the biometric authentication (after 2 failed attempts, if the user is wearing a mask or something) would trigger this error. there isn't a list of error codes provided by the keychain lib to exclude this error - there is only a hard coded string error displayed and it seems OS specific